public/javascripts/items.js

```diff
@@ -10,25 +10,83 @@ const fetchParams = {
 const params = completed =>
   Object.assign( {}, fetchParams, { body: JSON.stringify({ completed }) })
 
+const params2 = title =>
+  Object.assign( {}, fetchParams, { body: JSON.stringify({ title }) })
+
+const params3 = description =>
+  Object.assign( {}, fetchParams, { body: JSON.stringify({ description }) })
+
 $(document).ready( () => {
-  $( '.title' ).click( event => {
+  $('.updateTitle').keypress( event => {
```

Comment:
Whenever I have a number of event handlers to define on `document` load, I generally define the handlers elsewhere as functions, so that the code is a little easier to understand:
```js
const titleEdited = event => {
  // Code here ...
}

$( document ).ready( () => {
  $( '.edit-title' ).keypress( titleEdited )
  $( '.title > span' ).click( titleClicked )
  $( '.edit-description' ).keypress( descriptionEdited )
  $( '.description > span' ).click( descriptionClicked )
  $( '.completed' ).clicked( completedClicked )
})
```
Please note that class names should be kebab case: `update-title` instead of `updateTitle`.  I also propose a change to the `.checkmark` class name - we should select something that is relevant for our domain, and `completed` was the first term that came to mind for me - as well as the `updateTitle` and `updateDescription` classes.  Those last two seem to tell me what the back end is doing, not what the intent of the control is.